src/frontend/hooks/useSetHeader.tsx

```diff
@@ -0,0 +1,52 @@
+import { NavigationProp } from "@react-navigation/native";
```

Comment:
In react nav 6, the header can be set directly in the component. This custom hooks reduces the boiler plate in translations and in `useLayoutEffect`